### ISSUE TITLE
Ошибка SLF4J

### ISSUE THREAD

**Issue by Guap18** [Guap18/WEB#1]
# Описание
Тест не проходит

# Шаги воспроизведения

1. Запускаем тест на проверку
2. Тест валится
3. Смотрим на вывод в консоли 

* Ожидаемый результат - Тест прошёл успешно
* Фактический результат - Ошибка "No SLF4J providers were found."
# Скриншот 
<img width="617" alt="Снимок экрана 2024-02-18 в 13 52 24" src="https://github.com/Guap18/WEB/assets/137062261/b1843d39-9fb7-47bc-b0bf-1841abfc4c2a">

# Окружение 
* **Операционная система:** MacOS (Ventura 13.4)
* **IDE:** IntelliJ IDEA 2023.1.4 (Community Edition)
* **Java:** OpenJDK 11
* **Junit5**

**Comment by andrew-pahomov** [comment]
Это не ошибка, это информационное сообщение.

**Comment by Guap18** [comment]
Так, тогда я понять не могу, где и почему у меня не проходит тест
Зависимости указала

**Comment by Guap18** [comment]
<img width="973" alt="Снимок экрана 2024-02-18 в 15 17 27" src="https://github.com/Guap18/WEB/assets/137062261/f1dd2540-a21b-4a0f-b8ca-fcb2c415e360">
